frontend/src/settings/beta/ThemeTab.tsx

```diff
@@ -0,0 +1,152 @@
+import { Box, Button, Card, CardContent, CardHeader, Divider, Grid, Stack, Tab, Tabs, Theme } from '@mui/material'
+import { styled, ThemeProvider } from '@mui/material/styles'
+
+import { betaLightTheme, darkTheme, earthTheme, lightTheme, sunsetTheme } from '../../../src/theme'
+import { User } from '../../../types/types'
+
+interface ThemeTabProps {
+  user: User
+}
+
+interface ThemeKey {
+  name: string
+  theme: Theme
+}
+
+// TODO - determine how we are storing user themes
+
+export default function ThemeTab({ user }: ThemeTabProps) {
+  const themes: ThemeKey[] = [
+    { name: 'Bailo Default', theme: betaLightTheme },
+    { name: 'Bailo Classic', theme: lightTheme },
+    { name: 'Bailo Dark', theme: darkTheme },
+    { name: 'Bailo Sunset', theme: sunsetTheme },
+    { name: 'Bailo Earth', theme: earthTheme },
+  ]
+
+  function userThemeOnChange(themeKey) {
+    console.log(`Setting ${themeKey.name} as default theme for ${user.id}`)
+  }
+  return (
+    <Grid container direction='row' justifyContent='center' alignItems='center'>
+      {themes.map((theme) => (
+        <Grid key={theme.name} item xs={12} sm={4} spacing={2}>
+          <ExampleThemeDisplay themeKey={theme} setUserTheme={userThemeOnChange} />
+        </Grid>
+      ))}
+    </Grid>
+  )
+}
+
+interface TestThemeDisplayProps {
+  themeKey: ThemeKey
+  setUserTheme: (themeKey: ThemeKey) => void
```

Comment:
Instead of assuming intent with what this component will be used for, can we change it to `onThemeChange` to match our event/handler naming convention?